pkg/kubectl/cmd/util/helpers.go

```diff
@@ -573,6 +588,37 @@ func ChangeResourcePatch(info *resource.Info, changeCause string) ([]byte, types
 	}
 }
 
+// RemoveAnnotationsPatch craetes a patch between the origin input resource info
+// and the input resource info without change-cause annotation
+func RemoveAnnotationsPatch(info *resource.Info) ([]byte, types.PatchType, error) {
```

Comment:
To keep in-line with the name of related helpers, maybe rename to RemoveChangeResourcePatch